io.github.CatalystDevOrg.Catalyst/io.github.CatalystDevOrg.Catalyst.json

```
@@ -0,0 +1,64 @@
+{
+  "app-id": "io.github.CatalystDevOrg.Catalyst",
+  "base": "org.electronjs.Electron2.BaseApp",
+  "base-version": "23.08",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "23.08",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "run.sh",
+  "separate-locales": false,
+  "finish-args": [
+    "--share=ipc",
+    "--share=network",
+    "--socket=wayland",
+    "--socket=x11",
+    "--socket=pulseaudio",
+    "--device=dri",
+    "--talk-name=org.freedesktop.Notifications",
+    "--talk-name=org.kde.StatusNotifierWatcher",
+    "--talk-name=org.gtk.Notifications",
+    "--own-name=io.github.CatalystDevOrg.Catalyst"
+  ],
+  "modules": [
+    {
+      "name": "catalyst",
+      "buildsystem": "simple",
+      "build-commands": [
+        "ar x catalyst_3.5.4_amd64.deb",
+        "tar -xf data.tar.zst",
+        "cp -r usr/* /app/",
+        "mkdir -p /app/share/metainfo",
+        "mkdir -p /app/share/icons/hicolor/512x512/apps/",
+        "mv icon-512x512.png /app/share/icons/hicolor/512x512/apps/io.github.CatalystDevOrg.Catalyst.png",
+        "mv /app/share/applications/catalyst.desktop /app/share/applications/io.github.CatalystDevOrg.Catalyst.desktop",
+        "cp io.github.CatalystDevOrg.Catalyst.appdata.xml /app/share/metainfo/io.github.CatalystDevOrg.Catalyst.metainfo.xml",
```

```suggestion
        "install -Dm644 io.github.CatalystDevOrg.Catalyst.appdata.xml /app/share/metainfo/io.github.CatalystDevOrg.Catalyst.metainfo.xml",
```

but with the currently incorrect layout of the repo, this won't work.